test/format/t.c

```diff
@@ -348,7 +348,12 @@ main(int argc, char *argv[])
         wts_open(g.home, &g.wts_conn, true);
         timestamp_init();
     }
-
+    /*
+     * Since RTS is not ran with precise checkpoint, we need to use prepare discover cursor to claim
+     * all pending prepared transactions.
+     */
+    if (GV(PRECISE_CHECKPOINT))
```

Comment:
Here shouldnt we also check if ops.prepare is true?

Reply:
Adding a check for ops.prepare being true makes the code a bit clearer, but I’ll leave it to your judgment, it’s just a minor nit.

Reply:
done